Changelog Oct 31 (#3555)
timescale/docs: about/changelog.md

All the latest features and updates to Timescale products.

## 🎃 New AI, data integration, and performance enhancements
<Label type="date">October 31, 2024</Label>

> ### Pgai Vectorizer: Vector embeddings as database indexes (Early access)

Reviewer: ```suggestion
### Pgai Vectorizer: vector embeddings as database indexes
```

> Automatically create, update, and maintain embeddings as your data changes. Just like an index, Timescale handles all the complexity: syncing, versioning, and cleanup happen automatically.

Reviewer: ```suggestion
This early access feature enables you to automatically create, update, and maintain embeddings as your data changes. Just like an index, Timescale Cloud handles all the complexity: syncing, versioning, and cleanup happen automatically.
```

This means no manual tracking, zero maintenance burden, and the freedom to rapidly experiment with different embedding models and chunking strategies without building new pipelines.
Navigate to the AI tab in your service overview and follow the instructions to add your OpenAI API key and set up your first vectorizer.

![Vectorizer setup](https://s3.amazonaws.com/assets.timescale.com/docs/images/vectorizer-setup.png)

> 

Reviewer: I'd add a point here saying something along the lines of: 

Want to code vectorizers into your database by hand? Follow our [Vectorizer automation guide](https://github.com/timescale/pgai/blob/main/docs/vectorizer.md).

> ### PostgreSQL-to-PostgreSQL Foreign Data Wrappers: 

Reviewer: ```suggestion
### PostgreSQL-to-PostgreSQL with foreign data wrappers
```

> Fetch and query data from multiple PostgreSQL databases, including time-series data in hypertables, directly within Timescale using [Foreign Data Wrappers](https://docs.timescale.com/use-timescale/latest/schema-management/foreign-data-wrappers/). No more complicated ETL processes or external tools—just seamless integration right within your SQL editor. This feature is ideal for developers who manage multiple PostgreSQL and time-series instances and need quick, easy access to data across databases.

Reviewer: ```suggestion
Fetch and query data from multiple PostgreSQL databases, including time-series data in hypertables, directly within Timescale Cloud using [Foreign Data Wrappers](https://docs.timescale.com/use-timescale/latest/schema-management/foreign-data-wrappers/). No more complicated ETL processes or external tools—just seamless integration right within your SQL editor. This feature is ideal for developers who manage multiple PostgreSQL and time-series instances and need quick, easy access to data across databases.
```

### Chunk interval recommendations

> The Timescale console will now show recommendations for services with too many small chunks in their hypertable. Users who fit this criteria will see the recommendation on the corresponding service and on the corresponding hypertable(s), with a recommendation for the new interval. Users can then change their chunk interval within the UI to fix the change and improve their performance.

Reviewer: ```suggestion
Timescale Console now shows recommendations for services with too many small chunks in their hypertables. 
Recommendations for new intervals that improve service performance are displayed for each underperforming service and hypertable. Users can then change their chunk interval and boost performance within Timescale Console.
```
